tests/scripts/run_python_model_tests.sh

```diff
@@ -59,4 +59,11 @@ if [ "$ARCH_NAME" != "wormhole_b0" ]; then
 
     # Unet Shallow
     WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest -svv $TT_METAL_HOME/models/experimental/functional_unet/tests/test_unet_model.py
+
+    # Mamba
+    WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest -svv \
```

Comment:
No `-k` 😓 not sure if that's wasn't clear in my original comment

updated in original issue discussing this https://github.com/tenstorrent/tt-metal/issues/7586